cmd/tsconnect/src/lib/ssh.ts

```diff
@@ -27,26 +27,39 @@ export function runSSHSession(
 
   term.focus()
 
-  const sshSession = ipn.ssh(def.hostname, def.username, {
-    writeFn: (input) => term.write(input),
-    setReadFn: (hook) => (onDataHook = hook),
+  let resizeObserver: ResizeObserver | undefined
+  let handleBeforeUnload: ((e: BeforeUnloadEvent) => void) | undefined
+
+  const sshSession = ipn.ssh(def.hostname + "2", def.username, {
+    writeFn(input) {
+      term.write(input)
+    },
```

Comment:
Why has the structure of this changed in comparison to the existing implementation?